examples/notebooks/FastaiLRFinder_MNIST.ipynb

```diff
@@ -11,7 +11,49 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install --pre --upgrade pytorch-ignite tensorboardX"
```

Comment:
Maybe we can remove one of the identical cells and let's use stable version instead of a nightly one:
```
!pip install --upgrade pytorch-ignite tensorboardX
```
Let's also remove blank cells too.
What do you think ?

Reply:
Hi vfdev-5  thanks for the comment, I'll reflect it. Though I need more time to reflect the changes I've been bombarded with work lately. I'll finish the changes on the third weekend of december (17-19 dec)

Reply:
Hi Abdul, no worries, I asked @sdesrozis to finish up this PR. He can commit to this PR. Thanks again for your contributions.

Reply:
@vfdev-5 oh I didn't see the commits on github ios apps lol. I apologize for taking so long again and thank you for your guidance. You'll see me again soon looking for issues to fix :)